src/services/authService.js

```diff
@@ -50,4 +114,56 @@ export default class AuthService {
 
     return { token, ...Helper.omitFields(user, ['password']) };
   }
+
+  /**
+     *
+     * @param {Object} req query
+     * @returns {JSON} data
+     */
+  static async verify(req) {
+    const { token } = req.query;
+    const isExpire = Helper.verifyToken(token);
+
+    if (!isExpire) {
+      throw new Error('Expired Verification Link, resend verification Link');
+    }
+
+    const isUser = await User.findOne({ where: { id: isExpire.id } });
+
+    if (!isUser) throw new Error('User not find, please sign up');
+
+    if (isUser.dataValues.is_verified) throw new Error('User Email is Already Verified');
+
+    await User.update({ is_verified: true }, { where: { id: isExpire.id } });
+
+    return true;
```

Comment:
Instead of returning true, please return a message eg: `User successfully verified`